ament_cpplint/ament_cpplint/cpplint.py

```diff
@@ -6031,7 +6031,8 @@ def ProcessFile(filename, vlevel, extra_check_functions=[]):
                                         codecs.getwriter('utf8'),
                                         'replace').read().split('\n')
     else:
-      lines = codecs.open(filename, 'r', 'utf8', 'replace').read().split('\n')
+      with codecs.open(filename, 'r', 'utf8', 'replace') as f:
+          lines = f.read().split('\n')
```

Comment:
This file is imported from an external project. Please create a pull request there.

Reply:
Do you just want me to open it, or are you going to reject this change and wait for it to trickle down from upstream?

Reply:
There has been recent activity upstream so maybe it gets accepted. We don't have to wait for it though but it will reduce our divergence when we pull a new version in the future.